AnatomyOfMatplotlib-Part1-Figures_Subplots_and_layouts.ipynb

```diff
@@ -87,33 +90,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "If you are working interactively at an (I)python prompt, the GUI framework is not critical (mostly aesthetic) however when working in Jupyter we need to pick a backend that integrates with Jupyter (javascript) framework.\n",
```

Comment:
This is still useful content for people who will are reading this that will mostly work at a prompt.  Can it just be re-worrked rather than fully deleted?

Reply:
Added that part back. Hope that's fine.